src/shared/util/date-validation.ts

```diff
@@ -0,0 +1,10 @@
+import moment from 'moment'
+/**
+ * @param date
+ * @returns a moment with the date in the format 'DD MMM YYYY'
+ */
+export const createMomentFromDateString = (date: string): moment.Moment => {
```

Comment:
is there a reason this is in shared? Seems to be only used in dateValidator.ts and can probably be inlined there.

Reply:
shifted to inline